include/xgboost/tree_model.h

```diff
@@ -410,6 +410,8 @@ struct RTreeNodeStat {
   bst_float base_weight;
   /*! \brief number of child that is leaf node known up to now */
   int leaf_child_cnt;
+  /*! \brief sum instances statistics */
+  bst_uint sum_instance;
```

Comment:
Since the content of node statistics vector gets dumped to a binary file, we cannot add a new field `sum_instance` without breaking backward compatibility.